packages/rs-drive-abci/src/execution/fee_pools/fee_distribution.rs

```diff
@@ -875,7 +888,8 @@ mod tests {
                 proposer_payouts,
                 Some(ProposersPayouts {
                     proposers_paid_count: proposers_count,
-                    paid_epoch_index: 0
+                    paid_epoch_index: 0,
+                    refunded_epochs_count: 0
```

Comment:
I think it has nothing to do with payouts. Why not just to add to `ProcessedBlockFeesResult`?